pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -176,11 +176,14 @@ func (m *manager) AddContainer(p *v1.Pod, c *v1.Container, containerID string) e
 	cpus := m.state.GetCPUSetOrDefault(containerID)
 	m.Unlock()
 
-	err = m.updateContainerCPUSet(containerID, cpus)
-	if err != nil {
-		glog.Errorf("[cpumanager] AddContainer error: %v", err)
-		return err
+	if !cpus.IsEmpty() {
```

Comment:
Could you add a debug log line here indicating why the update call is skipped in this case? Future policies might not be as simple as the two we currently have.

Reply:
Good point!